### ISSUE TITLE
Feature Request: Better workflow for testing prompt stability

### ISSUE THREAD

**Issue by aantn** [promptfoo/promptfoo#428]
Promptfoo is very good at comparing different prompts across a set of test cases. However, when I test prompts I usually want to know not just "is this prompt better than this prompt when run once", but also "will I consistently get good results from this prompt  vs that prompt".

Today I set evaluateOptions.repeat=5 (or some other low number where I can manually review results) and compare the responses. So for each testcase X prompt I review 5 rows and compare them manually.

Ideally, the repeated results would appear in a single tablerow and give me some indication if I got identical responses. (E.g. allow navigating between different responses with left/right arrows on each cell. If a response was received multiple times, show it only once and list how many times each response was received.)

It would also be nice to have some form of automated eval re. response similarity and/or an AI explanation of the differences between them.

I'm dreaming a little on this, but I wanted to share how I'm using Promptfoo today and how it could be more useful for me. If there are other workflows/solutions for this sort of thing, would love to hear.

**Comment by aantn** [comment]
One more comment on this: the "Show different only" view is very nice, but it's for comparing prompts not repeat runs on the same prompt.

**Comment by Kain-90** [comment]
µI have the same idea, i'm willing to submit a PR, but i'm not familiar with javascript.

**Comment by aantn** [comment]
I'm closing this, as its a bit too "general". I'll open issues with more specific ideas for improvement, starting with this.

https://github.com/promptfoo/promptfoo/issues/992
